src/server/models/match.model.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+let matchSchema = new Schema({
+  mentorId: Schema.Types.ObjectId,
```

Comment:
Just so I understand how this works: matchSchema here and matchData above don't seem to have the same set of fields. How does this work then? On line 14 of match.controller.js, matchData seem to be instantiating from matchSchema. 

For this to work, we'd have to store things like:

matchData.mentorId = req.mentorId // get the mentorId from request
matchData.menteeId = req.session.user_id // get the current session's user ID.
... etc.

For now, you can just add a TODO and remove lines 65,66, 76, 77 from match.controller.js since those methods aren't complete yet.